examples/js/bots/liquidity_bot.ts

```diff
@@ -0,0 +1,66 @@
+import { defaultClient } from "../client";
+import { ORDER_SIDE_ASK, ORDER_SIDE_BID, ORDER_TYPE_LIMIT, TestUser } from "../config";
+
+const base = "DIF";
+const quote = "BTC";
+const market = `${base}_${quote}`;
+const botUseFunds = 0.8; // Use 80% of funds
+const deviation = 1.002;
+const tiersAmount = 10;
+const fee = "0";
+const balanceCache = {};
+
+const tick = async (user_id: TestUser) => {
+  await defaultClient.connect();
+  const balance = await defaultClient.balanceQuery(user_id);
+
+  const Q_BAL = (+balance.get(quote).available + +balance.get(quote).frozen) * botUseFunds;
+  const B_BAL = (+balance.get(base).available + +balance.get(base).frozen) * botUseFunds;
```

Comment:
Note that these are the only consts you use SCREAMING_SNAKE_CASE for. Why not also use camelCase? --> `quoteBalance` and `baseBalance` for example

As far as I can see only imported consts are SCREAMING_SNAKE_CASE.